pandas/io/excel/_openpyxl.py

```diff
@@ -8,6 +8,9 @@
 from pandas.io.excel._base import ExcelWriter, _BaseExcelReader
 from pandas.io.excel._util import _validate_freeze_panes
 
+if TYPE_CHECKING:
+    from openpyxl.descriptors.serialisable import Serialisable as StyleObject
```

Comment:
Is there a reason for the alias here? I think clearer if just left as Serialisable

Reply:
I'll change it back.